x-pack/plugins/observability/common/custom_threshold_rule/helpers/get_group.ts

```diff
@@ -22,10 +22,8 @@ export const getGroupQueries = (
   return (
     (groups &&
       groups.map((group) => ({
-        term: {
-          [groupFieldName || group.field]: {
-            value: group.value,
-          },
+        match_phrase: {
```

Comment:
@maryam-saeidi why do we need `match_phrase` instead of `term`?

Reply:
Filters use `match_phrase` by default, so I used them here, too.